eberron.md

```diff
@@ -188,9 +197,9 @@ Add a wound die for every level cast above cantrip.
 - **Range:** touch
 - **Duration:** permanent
 
-Causes a piece of wood no heavier than 5 pounds to be shaped according to your thoughts.
+Causes a piece of wood no heavier than 40 pounds to be shaped according to your thoughts.
```

Comment:
5 lbs of wood is a very small amount of wood. This along with doubling every level puts it in line with levitate. You can play with the numbers, but somewhere between 10-40 lbs makes the most sense I think. Depends on how intricate you want the shaping to be.